.github/workflows/cd.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
       - name: Run Unit Tests
         working-directory: src/PollinationSDK.Tests
         run: |
+          $Env:PollinationApiKey = ${{secrets.POLLINATION_STAGING_APIKEY}}
```

Comment:
Hi @AntoineDao, currently I generate this ApiKey from my account on staging server. It seems working well for running unit tests. But it expires very quick. Can we have an option for setting up the expire date or have a developer's ApiKey that doesn't expire (maybe expire every 12 months)?

Reply:
The API key never expires so I am not sure what is going wrong here 😅